pkg/services/sso/keycloak_service_builder.go

```diff
@@ -101,12 +101,20 @@ func build(providerName string, keycloakConfig *keycloak.KeycloakConfig, realmCo
 	if providerName == keycloak.MAS_SSO ||
 		realmConfig != nil {
 		_, realmConfig := arrays.FindFirst(notNilPredicate, realmConfig, keycloakConfig.KafkaRealm)
-		return newKeycloakService(keycloakConfig, realmConfig.(*keycloak.KeycloakRealmConfig))
```

Comment:
is this func used anywhere? If not it can be deleted.

Reply:
Good point. Was used by one test, but I replaced it with the builder. Removed.